README.md

```diff
@@ -43,14 +43,21 @@ A minimalist but powerful productivity timer designed to keep you focused and fr
 - Insistent notifications
 - All future features for free
```

Comment:
Could you please delete this row too?
I copy-pasted it from the Google Play description, where it's part of the "Pro version" and it doesn't make sense here.

Reply:
No worries, it's done.